composer.json

```
@@ -20,27 +20,26 @@
         "neos/neos": "7.1.x-dev",
         "neos/site-kickstarter": "7.1.x-dev",
 
-        "neos/demo": "@dev",
-        "neos/neos-ui": "@dev",
-        "neos/seo": "~3.0",
-        "neos/redirecthandler-neosadapter": "~4.0",
-        "neos/redirecthandler-databasestorage": "~4.0",
-        "neos/redirecthandler-ui": "~2.0",
-
-        "neos/setup": "@dev",
-        "neos/neos-setup": "@dev",
+        "neos/demo": "^7.1",
+        "neos/neos-ui": "^7.1",
+        "neos/seo": "^3.0",
+        "neos/redirecthandler-neosadapter": "^4.3",
+        "neos/redirecthandler-databasestorage": "^5.0",
+        "neos/redirecthandler-ui": "^2.4",
+        "neos/setup": "^5.0",
+        "neos/neos-setup": "^1.1",
         "neos/content-repository": "7.1.x-dev",
         "neos/fusion": "7.1.x-dev",
         "neos/media": "7.1.x-dev",
         "neos/media-browser": "7.1.x-dev",
         "neos/diff": "7.1.x-dev"
     },
     "require-dev": {
-        "neos/buildessentials": "@dev",
+        "neos/buildessentials": "^7.0",
         "mikey179/vfsstream": "^1.6",
         "phpunit/phpunit": "^9.0",
-        "symfony/css-selector": "~2.0",
-        "neos/behat": "@dev"
+        "symfony/css-selector": "^2.0",
+        "neos/behat": "^6.1"
```

```suggestion
        "neos/behat": "^7.1"
```
We branch off behat with the other packages now: https://github.com/neos/behat/tree/7.1
See https://github.com/neos/flow-development-distribution/pull/63

---

Agreed ... I think that those scripts should not touch dependencies that do not have to change change. 

In general it would imho even be ok if the base-dist would only work with released branches of neos i am pretty sure none uses the base-dist as dev-master.